0_portfolio_test.R

```diff
@@ -9,6 +9,12 @@ get_ald_scen <- function(portfolio_type){
     
     ald <- read_rds(paste0(analysis_inputs_path, "/equity_ald_scenario.rda"))
     
+    # fix naming issue. if equity market list is either Global or GlobalMarket, allow both
+    # values. Else retain the input from the config file
+    equity_market_list <- ifelse(c("Global", "GlobalMarket") %in% equity_market_list,
```

Comment:
Does this function work in that, if one of Global or GlobalMarket is in the list then they are both added? Or does it work that they both have to be in the list?

Reply:
if either one is in the list, both are added, so that it doesn't matter anymore which one to choose. Multiple people ran into issues with this over the last few weeks (that's also what caused the problem at the Norwegian financial supervision btw) and this has solved it for me every time.